src/Fantomas/CodePrinter.fs

```diff
@@ -894,7 +894,7 @@ and genExpr astContext synExpr =
             +> ifElse isArrow (sepArrow +> preserveBreakNln astContext e2) (!- " do" +> indent +> sepNln +> genExpr astContext e2 +> unindent))
 
     | CompExpr(isArrayOrList, e) ->
-        let astContext = { astContext with IsNakedRange = true }
+        let astContext = { astContext with IsNakedRange = false }
```

Comment:
Maybe removing this line would be clearer and with same effect?

Reply:
Yep, it has the same effect.